### ISSUE TITLE
Failed to extract version bump info from commit message

### ISSUE THREAD

**Issue by simenandre** [mAAdhaTTah/combine-dependabot-prs#3]
Hello 👋

Thank you for creating this package!

I'm trying to run this command:
`npx combine-dependabot-prs bjerkio/website`

This is the output of that command:

```shell
ℹ Cloning repository bjerkio/website into /var/folders/rj/fws4wktx5nn6yy4rldhs19_80000gn/T/cdp-kWnoZk/bjerkio--website.
ℹ Setting up repository for committing.
ℹ Combining PRs in repo bjerkio/website.
⚠ Failed to extract version bump info from commit message: Merge branch 'main' into dependabot/npm_and_yarn/gatsby-plugin-image-1.8.0
⚠ Failed to extract version bump info from commit message: chore(deps): bump gatsby-plugin-robots-txt from 1.6.2 to 1.6.8
⚠ Failed to extract version bump info from commit message: chore(deps): bump gatsby-plugin-offline from 4.7.1 to 4.8.0
⚠ Failed to extract version bump info from commit message: chore(deps): bump @theme-ui/color-modes from 0.9.1 to 0.10.0
⚠ Failed to extract version bump info from commit message: chore(deps): bump react-i18next from 11.10.0 to 11.11.0
⚠ add-darkmode does not start with dependabot. Not combining.
```

The pull requests that fails are bjerkio/website#210 bjerkio/website#209 bjerkio/website#208 bjerkio/website#211

I haven't dived into your code yet too see if I understand why this is happening, but wanted to let you know. I'd gladly help out! 👍

**Comment by mAAdhaTTah** [comment]
Thanks for the bug report! Fixed in v1.0.4.

**Comment by simenandre** [comment]
Wow! That was quick! 😳

Thank you so much! 🙏
